pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -204,6 +217,42 @@ func (pl *CSILimits) filterAttachableVolumes(
 	return nil
 }
 
+// checkAttachableInlineVolume takes an inline volume and add to the result map if the
+// volume is migratable and CSI migration for this plugin has been enabled.
+func (pl *CSILimits) checkAttachableInlineVolume(vol v1.Volume, csiNode *storagev1.CSINode,
+	pod *v1.Pod, result map[string]string) error {
+	if isMigratable := pl.translator.IsInlineMigratable(&vol); !isMigratable {
+		return nil
+	}
+	// Check if the intree provisioner CSI migration has been enabled.
+	inTreeProvisionerName, err := pl.translator.GetInTreePluginNameFromSpec(nil, &vol)
+	if err != nil {
+		return fmt.Errorf("error looking up provisioner name for volume: %v. Error Msg: %v", vol, err)
+	}
+	if !isCSIMigrationOn(csiNode, inTreeProvisionerName) {
+		klog.V(5).InfoS("CSI Migration of provisioner is not enabled", "provisioner", inTreeProvisionerName)
+		return nil
+	}
+	// Do translation for the in-tree volume.
+	translatedPV, err := pl.translator.TranslateInTreeInlineVolumeToCSI(&vol, pod.Namespace)
+	if err != nil || translatedPV == nil {
+		return fmt.Errorf("Cannot convert volume(%v) from inline to csi.", vol)
```

Comment:
Error have different rules :sweat_smile: 

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

```suggestion
		return fmt.Errorf("converting volume(%v) from inline to csi: %w", vol, err)
```

Reply:
Although you should probably have a different error message for `translatedPV == nil`

Reply:
Done